### PR TITLE
CN translations via Gitlocalize for review 

### PR DESCRIPTION
**Pull request by pablanopete**
First PR from new tooling - trialing out the section please leave any comments.

### CODE REVIEW

**Comment by pablanopete** [comment]
@laboon @lsaether can you make sure this is correct and doesn't break anything :)

**Comment by laboon** [comment]
You got it @pablanopete

File: zh/docs/faq.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+## 什么是预警网络？
+
+Kusama作为波卡的预警网络，通过提供对一些事务的预警从而保障开发者的安全。如果没有像Kusama这样的网络，就没有合理的方法充分理解今后潜在的危险。我们在构建最尖端的试验性技术。也就是说，没有办法保证Kusama会做什么，或者它将如何运作——敬请期待许多混乱（和一些乐趣）
+
+很明显Kusama不是一个测试网。它是一个早期的，高度试验性的波卡版本，且具有实际的经济场景。这个网络属于社区，不会因为中心化的主网切换而消失。只要有社区的维护，它就会继续存在下去。我们设想它会适合那些新颖的，早期的功能，也适合那些准备在波卡上开发和部署的项目。
+
+## 使用Kusama能得到什么奖励？
+
+波卡创世时1%的DOT（Web3基金会的100000个DOT）将被留作可能的激励，发放给Kusama的持币人和社区。具体的机制还没有被决定，不过会在接近创世的时候确定。
```
Comment (ansonla3) [inline zh/docs/faq.md]:
```suggestion
波卡创世时1%的DOT（Web3基金会的100,000个DOT）将被留作可能的激励，发放给Kusama的持币人和社区。具体的机制还没有被决定，不过会在接近创世的时候确定。
```

File: zh/docs/faq.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# 常见问题
+
+## 什么是Kusama
+
+Kusama是一个早期的，未经审计和优化的波卡发行版本。Kusama作为一个试验场，项目方和开发者可以在这里开发部署平行链，或者在真实环境中尝试波卡的治理，质押，提名和验证功能。
```
Comment (ansonla3) [inline zh/docs/faq.md]:
```suggestion
Kusama是一个早期的，未经审计和优化的波卡发行版本。Kusama作为一个试验场，项目方和开发者可以在这里真实环境中开发部署平行链，或者试用波卡的治理，质押(抵押)，提名和验证功能。
```

File: zh/docs/faq.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+## 什么是预警网络？
+
+Kusama作为波卡的预警网络，通过提供对一些事务的预警从而保障开发者的安全。如果没有像Kusama这样的网络，就没有合理的方法充分理解今后潜在的危险。我们在构建最尖端的试验性技术。也就是说，没有办法保证Kusama会做什么，或者它将如何运作——敬请期待许多混乱（和一些乐趣）
+
+很明显Kusama不是一个测试网。它是一个早期的，高度试验性的波卡版本，且具有实际的经济场景。这个网络属于社区，不会因为中心化的主网切换而消失。只要有社区的维护，它就会继续存在下去。我们设想它会适合那些新颖的，早期的功能，也适合那些准备在波卡上开发和部署的项目。
```
Comment (ansonla3) [inline zh/docs/faq.md]:
```suggestion
很明显Kusama不是一个测试网。它是一个早期的，高度试验性的波卡版本，且具有实际的经济场景。这个网络属于社区，不会因为中心化的主网切换而消失。只要有社区的维护，它就会继续存在下去。我们预见它会提供新颖，早期的功能和那些准备在波卡上开发和部署的项目。
```

File: zh/docs/faq.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+## 使用Kusama能得到什么奖励？
+
+波卡创世时1%的DOT（Web3基金会的100000个DOT）将被留作可能的激励，发放给Kusama的持币人和社区。具体的机制还没有被决定，不过会在接近创世的时候确定。
+
+此外也会有为重大问题准备的奖励计划。你可以在[这里](./break/bug-bounty.md)得到更多消息。
```
Comment (ansonla3) [inline zh/docs/faq.md]:
```suggestion
此外也会有为重大问题准备的赏金计划。你可以在[这里](./break/bug-bounty.md)得到更多消息。
```

File: zh/docs/faq.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
+# 常见问题
+
+## 什么是Kusama
+
+Kusama是一个早期的，未经审计和优化的波卡发行版本。Kusama作为一个试验场，项目方和开发者可以在这里开发部署平行链，或者在真实环境中尝试波卡的治理，质押，提名和验证功能。
+
+## 什么是预警网络？
```
Comment (ansonla3) [inline zh/docs/faq.md]: would 金丝雀网络 seem better?

Comment (Satoshi-Kusumoto) [reply zh/docs/faq.md]: It depends. But I will change it to 金丝雀网络

**Comment by Satoshi-Kusumoto** [comment]
I do not have write access to neither this repo nor the fork. So I am starting a PR from the fork. Maybe next time the review should be moved to gitlocalize or we figure out another better way

**Comment by Satoshi-Kusumoto** [comment]
pablanopete/userguide#2

**Comment by pablanopete** [comment]
Yes I think from here on we'll do reviews in gitlocalize, this was my mistake that I put into a PR before on the fork.

**Comment by pablanopete** [comment]
merged it in my fork looks like it is reflecting here as well, anything further needed from my end @ansonla3 @laboon ?

**Comment by Satoshi-Kusumoto** [comment]
@ansonla3 I applied all of your suggestions, except that staking is commonly translated into 质押 but not 抵押

**Comment by ansonla3** [comment]
@Satoshi-Kusumoto imo we can still keep that using "()" as a reference based on the resources I have read in the below:

https://www.chainnews.com/articles/961697664985.htm
https://xw.qq.com/cmsid/20190705A0M08L00
https://www.eoscanada.com/zh/blog/%E4%BB%80%E4%B9%88%E6%98%AFstaking%E6%8A%B5%E6%8A%BC%E5%92%8C-unstaking%E5%8F%96%E6%B6%88%E6%8A%B5%E6%8A%BCeos
https://www.bilingling.com/news/142541/
https://blockcast.it/2019/06/09/how-to-be-incentivized-by-staking-economy/

**Comment by Satoshi-Kusumoto** [comment]
@ansonla3 A reasonable suggestion. I added it pablanopete/userguide#3 @pablanopete

**Comment by pablanopete** [comment]
@Satoshi-Kusumoto merged; thanks for all the help.

**Pull request merged**